.github/workflows/fedora.yml

```diff
@@ -0,0 +1,27 @@
+name: Test on Fedora
+on:
+  - push
+
```

Comment:
It didn't tested on pull request. How about to add `pull_request:` at `on:` ?

Reply:
Done.